.gitignore

```diff
@@ -0,0 +1,2 @@
+*~
```

Comment:
If *your* editor does it, it should be in *your* personal gitignore. Otherwise you can have it in all of your repositories. I have configured my *vim* to save it to `~/.vim/backup` to not spam the file system with bogus files.

But whatever, I merged it.